app/qml/MMMapThemePanel.qml

```diff
@@ -0,0 +1,207 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+
+import lc 1.0
+import "."
+import "./components"
+
+Drawer {
+  id: root
+
+  visible: false
+  modal: true
+  interactive: false
+  dragMargin: 0 // prevents opening the drawer by dragging.
+
+  padding: 20
+
+  width: ApplicationWindow.window.width
+  height: ApplicationWindow.window.height / 2
```

Comment:
Should this be dynamic (e.g. what if you have massive screen and just 2 themes?) with appwindow.heigh/2 as max?